src/main/docker/artemis/Dockerfile

```diff
@@ -0,0 +1,56 @@
+#-----------------------------------------------------------------------------------------------------------------------
+# build stage
+#-----------------------------------------------------------------------------------------------------------------------
+FROM eclipse-temurin:17-jdk as builder
+
+WORKDIR /opt/artemis
+# so far just using the .dockerignore to define what isn't necessary here
+COPY . .
+
+RUN ./gradlew -i --stacktrace --no-daemon -Pprod -Pwar clean bootWar
+
+#-----------------------------------------------------------------------------------------------------------------------
+# runtime stage
+#-----------------------------------------------------------------------------------------------------------------------
+FROM eclipse-temurin:17-jre as runtime
+
+# Docker Compose: wget (healthcheck docker compose)
+# Artemis: graphviz, locales
+# versions just pinned for reproducibility
+RUN echo "Installing needed dependencies" \
+  && apt-get update && apt-get install -y --no-install-recommends locales=2.35-0ubuntu3.1 \
+  graphviz=2.42.2-6 wget=1.21.2-2ubuntu1 netcat-openbsd=1.218-4ubuntu1 \
```

Comment:
The base image whatever `eclipse-temurin:17-jdk` uses might update at any time (therefore it’s not reproducible anyway) which then probably breaks installing those specific versions.

Reply:
The version pinning was suggested by codacy. I can also remove it again since we are also not really pinning anywhere else.

Reply:
I would remove the pinning as well. Otherwise, the maintenance efforts are quite high as you have to check these deps for updates manually.

Reply:
K thx already removed the pinning.